src/Confluent.Kafka/Timestamp.cs

```diff
@@ -72,6 +71,15 @@ public override int GetHashCode()
         public static bool operator !=(Timestamp a, Timestamp b)
             => !(a == b);
 
+        /// <summary>
+        ///     Convert a DateTime instance to a unix timestamp.

```

Comment:
It's already in the name of function, but maybe specify it is millisecond and not second. Also precise in remarks that it converts in Utc before - local DateTime with unspecified Kind will be false